libr/search/search.c

```diff
@@ -42,6 +50,7 @@ R_API RSearch *r_search_new(int mode) {
 R_API RSearch *r_search_free(RSearch *s) {
 	if (!s) return NULL;
```

Comment:
See coding style and be consistent. This has been discussed zillions of times and im not gonna explain the reasons again